go.mod

```diff
@@ -42,4 +42,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
 )
 
-replace github.com/sirupsen/logrus => github.com/projectcalico/logrus v1.0.4-calico
+replace (
+	github.com/projectcalico/api => github.com/song-jiang/api v0.1.2-0.20210611091848-231bd75c074d
```

Comment:
We will update this pin once https://github.com/projectcalico/api/pull/3 get merged.